main.go

```diff
@@ -50,6 +51,12 @@ var (
 			Value:       "rancher/k3k:k3k-kubelet-dev",
 			Destination: &sharedAgentImage,
 		},
+		cli.StringFlag{
+			Name:        "shared-agent-pull-policy",
+			EnvVar:      "SHARED_AGENT_PULL_POLICY",
+			Usage:       "K3K Virtual Kubelet image pull policy",
+			Destination: &sharedAgentImagePullPolicy,
+		},
```

Comment:
Since this is an enum, I would put at least a quick check of the provided value. With `urfave/cli/v2` there is a Flag Action we can use, but for now we can stick for a manual check.

Reply:
we have a validation function, we should implement some kind of conditions to make sure we have a the right values